### PR TITLE
arbotix_python for ROS Noetic

### PR DESCRIPTION
**Pull request by calismurat**
Hi Mike, yet another copy of arbotix_python working with ROS Noetic. Please give it a try. Tested arbotix_python and arbotix_terminal. I didn't touch other packages inside arbotix_ros besides the one mentioned above.

### CODE REVIEW

File: arbotix_python/CHANGELOG.rst [hunk 1]
```diff
@@ -2,6 +2,13 @@
 Changelog for package arbotix_python
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+0.11.0 (2020-12-25)
```
Comment (mikeferguson) [inline arbotix_python/CHANGELOG.rst]: please revert the changes to the CHANGELOG - this is automatically generated when I do the release.

File: arbotix_python/package.xml [hunk 2]
```diff
@@ -1,6 +1,6 @@
-<package>
+<package format="3">
   <name>arbotix_python</name>
-  <version>0.10.0</version>
+  <version>0.11.0</version>
```
Comment (mikeferguson) [inline arbotix_python/package.xml]: please revert this one line

**Review by mikeferguson — CHANGES_REQUESTED** [review]
This looks good - I think the one other thing we probably need to do is update the shebangs to to python3. Please also revert the couple of changelog/version things - those get automatically done as part of the release process.

File: arbotix_python/CHANGELOG.rst (hunk 1 above)
Comment (calismurat) [reply arbotix_python/CHANGELOG.rst]: done.

File: arbotix_python/package.xml (hunk 2 above)
Comment (calismurat) [reply arbotix_python/package.xml]: done.

**Comment by calismurat** [comment]
> This looks good - I think the one other thing we probably need to do is update the shebangs to to python3. Please also revert the couple of changelog/version things - those get automatically done as part of the release process.

Ouch - reverted the autostuff. I would leave the shebangs as is until somebody posts an issue :D. It worked with python 2, now works with python 3. If you prefer to explicitly name the version, give me a sign, I'll change it.

**Review by mikeferguson — APPROVED** [review]

**Comment by mikeferguson** [comment]
Thanks for the PR and quick response to review comments.
